assets/scss/modules/_forms.scss

```diff
@@ -57,4 +58,33 @@
       transform: rotate(45deg);
     }
   }
+}
+
+.holder {
+  &-group {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+  }
+
+  &-size1of1 {
+    flex: 1;
+
+    &:nth-last-child(-n+3) {
+      margin-top: 30px;
+    }
+  }
+
+  &-col {
+    flex-wrap: nowrap;
+    flex-direction: row;
+  }
+
+  &-size1of2 {
+    flex: 1;
+
+    &:nth-child(1) {
+      margin-right: 4%;
```

Comment:
Strange measure warning!

Reply:
What is the problem?

Reply:
May be use ´justify-content: space-between;´